Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -257,8 +257,10 @@ lineWidth : expression;
 
 
 // 5.4.5.8   Print Statement
+//We make the part regarding the file handle optional to support VB6's print statement in forms.
+//It is an invocation of the Print member of the enclosing form, which also takes an output list as argument.
 printMethod : PRINT;
-printStmt : PRINT whiteSpace markedFileNumber whiteSpace? COMMA (whiteSpace? outputList)?;
+printStmt : printMethod (whiteSpace markedFileNumber whiteSpace? COMMA)? (whiteSpace? outputList)?;
```

Comment:
There is a separate `printStmt` rule.

Reply:
@bclothier I read this as simply taking a lexer token and making it a parser rule; there is no material change/impact on the parsing itself, just on the resulting parse trees. `PrintStmtContext` still refers to the "print to file" statement, only now with this change it'll have a `printMethod()` member that will yield a `PrintMethodContext` with a `PRINT()` method yielding the token, instead of a `PRINT()` member yielding the token directly.

Reply:
Since I realize now that this is about the `printStmt`. 
You can easily tell them apart by the presence of `markedFileNumber`.